packages/web-client/app/templates/card-pay/balances.hbs

```diff
@@ -8,10 +8,12 @@
           <CardPay::DashboardPanel::Section @section={{section}}>
             <:cta>
               <Boxel::Button
-                {{on "click" (set this "flow" section.workflow)}}
                 @kind="primary"
                 @size="touch"
                 @disabled={{section.isCtaDisabled}}
+                @as="link-to"
```

Comment:
I figured it would be cleaner to use `LinkTo`s (now when there's more query params) and deal with query params in route hooks.

Reply:
I think moving back to actions approach is likely better.

Reply:
Sure, will change back to use that.